aten/src/ATen/core/ivalue.cpp

```diff
@@ -118,4 +118,28 @@ void ivalue::Object::resizeObject(size_t slot) {
   slots_.resize(type()->numAttributes());
 }
 
+static bool CompareIValue(const std::pair<IValue, IValue>& aWrap,
```

Comment:
This same code is in `pickler.cpp` in `IValuePairComparator`, you could remove that and make it reference this or the other way around but either way there should only be 1.

Reply:
My bad. I plead ignorance. Happy to do it either way.